### ISSUE TITLE
Add "Reload"-Button

### ISSUE THREAD

**Issue by sistein** [threema-ch/threema-web#545]
## Expected Behavior
There should be a possibility to reload the page if there is no option provided by the browser.

## Current Behavior
Right now I only have the possibility to either restart the browser or "inspect" the webpage and reload it via the developer tools.

## Possible Solution
Add a button. 

## Steps to Reproduce (for bugs)

1. Install Vivaldi
2. Use Threema Web and add it to the left site.
3. Let it fail (like it is doing a lot of times right now)
4. Try to get it working again.
![grafik](https://user-images.githubusercontent.com/13909466/43632231-ceb959f6-9705-11e8-955f-7b01260f1208.png)

## Your Environment

* Threema Web version: 2.0 (but happened before) 
* Threema App version:3.5.2000429
* Browser name and version: Vivaldi | 1.15.1147.55 (Stable channel) (32-Bit)

* Computer operating system and version: Win 10 64 bit
* Smartphone operating system and version: Samsung Note 8 - fully updated

**Comment by SirTyson** [comment]
Mind if I work on this?

**Comment by dbrgn** [comment]
@SirTyson feel free!

Maybe the layout needs some tuning, it's already a bit crowded.

If you want, you can also use the occasion to remove that red warning box. We will probably remove it anyways starting the next release.

**Comment by lgrahl** [comment]
Adding a reload button feels a bit like fixing a design flaw of the browser. I would rather propose a way to retry the connection instead of reloading the whole page.

**Comment by dbrgn** [comment]
@lgrahl I agree, but adding a button would be an acceptable first step that can be improved later on, by changing the button behavior from reload to reconnect. Or do you think it's not worth the effort?

**Comment by lgrahl** [comment]
I think having a *retry* button that appears a few seconds once the loading indicator stalled for maybe 5 seconds is worth it. Let it behave like a reload for the beginning. Later, fix that behaviour to reconnect instead.
